### PR TITLE
Make it easier to find network modules

### PR DESCRIPTION
**Pull request by gundalow**
##### SUMMARY
Feedback has been it's difficult (via Google or directly) to find
modules as some people search for the company name vs product name,
therefore specify both.

##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
asa_command
fortios_config

##### ANSIBLE VERSION
```
2.4.0
```

### CODE REVIEW

File: lib/ansible/modules/network/iosxr/iosxr_config.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
 module: iosxr_config
 version_added: "2.1"
 author: "Ricardo Carrillo Cruz (@rcarrillocruz)"
-short_description: Manage Cisco IOS XR configuration sections
+short_description: Manage Cisco IOS-XR configuration sections
```
Comment (Qalthos) [inline lib/ansible/modules/network/iosxr/iosxr_config.py]: I'm not super sure, but everything I've seen seems to indicate the name is not actually hyphenated

Comment (gundalow) [reply lib/ansible/modules/network/iosxr/iosxr_config.py]: Good spot
hum, so it's `NX-OS` http://www.cisco.com/c/en/us/products/ios-nx-os-software/nx-os-software/index.html

But `IOS XR` http://www.cisco.com/c/en/us/support/ios-nx-os-software/ios-xr-software/tsd-products-support-series-home.html

**Comment by gundalow** [comment]
Sam and privateip have given this the nod, so merging

The pylint errors are not related.

**Comment by gundalow** [comment]
Merged into `devel` and `temp-staging-post2.3.0` (so it will end up in `stable-2.3`)

**Pull request merged**